torch_xla/csrc/ops/ops.cpp

```diff
@@ -692,7 +692,22 @@ torch::lazy::NodePtr Gelu(const torch::lazy::Value& input) {
   auto lower_fn = [](const XlaNode& node,
                      LoweringContext* loctx) -> XlaOpVector {
     xla::XlaOp xla_input = loctx->GetOutputOp(node.operand(0));
-    return node.ReturnOp(BuildGelu(xla_input), loctx);
+
+    // Building composite computation.
+    const std::string name = "composite.gelu";
+    const std::string attr = "{approximate = \"none\"}";
+    xla::XlaBuilder builder(name);
+    xla::XlaOp arg = xla::Parameter(
+        &builder, 0, ShapeHelper::ShapeOfXlaOp(xla_input), "arg");
+    xla::XlaOp ret = BuildGelu(arg);
+    xla::XlaComputation computation = ConsumeValue(builder.Build(ret));
+
+    // Building call to computation.
+    std::vector<xla::XlaOp> inputs{xla_input};
+    xla::XlaOp output = xla::CompositeCall(loctx->builder(), computation, inputs, name,
+                                           attr, /*version=*/1);
```

Comment:
ditto for version

Reply:
Yes this is for testing, I learned this setting from this [XLA UT](https://github.com/openxla/xla/blob/main/xla/hlo/builder/xla_builder_test.cc#L345-L348). I can remove it if it makes no scense.